### DISCUSSION TITLE
AUTH + SIGN SDK work together in one session

### DISCUSSION THREAD

**Discussion by chendinh** [WalletConnect/walletconnect-monorepo#2530] [Developer Support]
I wonder, is there any ways to use both auth sdk and sign sdk in one session ? 
now web3wallet-sdk can support both sdk, there are any dapp support both auth and sign sdk ? or it must be separated. If dapp can choose Auth sdk or Sign sdk or Both for all workflow ?
Thank you

**Comment by finessevanes** [comment]
Hey Chen, typically dapps support either auth or sign. Can you help me understand the use case for a wallet connecting to both an auth dapp and a sign dapp within the same session?
